geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ConfigurePDXCommand.java

```diff
@@ -96,59 +72,56 @@ public Result configurePDX(
 
     // Set persistent and the disk-store
     if (diskStore != null) {
-      cache.setPdxPersistent(true);
-      ird.addLine(CliStrings.CONFIGURE_PDX__PERSISTENT + " = " + cache.getPdxPersistent());
-
-      if (!diskStore.equals("")) {
-        cache.setPdxDiskStore(diskStore);
-        ird.addLine(CliStrings.CONFIGURE_PDX__DISKSTORE + " = " + cache.getPdxDiskStore());
-      } else {
-        ird.addLine(CliStrings.CONFIGURE_PDX__DISKSTORE + " = " + "DEFAULT");
-      }
+      ird.addLine(CliStrings.CONFIGURE_PDX__PERSISTENT + " = true");
```

Comment:
A nitpick, but I'd rename `ird` to something more meaningful (even if it's just `result` or `infoResultData`) while you're here.

Reply:
Done